src/bindgen/ir/item.rs

```diff
@@ -21,6 +22,8 @@ pub trait Item {
 
     fn container(&self) -> ItemContainer;
 
+    fn populate_ctyperesolver(&self, _resolver: &mut CTypeResolver) { unimplemented!() }
+    fn set_ctype(&mut self, _resolver: &CTypeResolver) { unimplemented!() }
```

Comment:
Along with the rename of ctype above, I think changing `populate_ctyperesolver` to `add_declaration_types` or `collect_declaration_types` and `set_ctype` to `resolve_declaration_types` would match the other functions here and might be clearer.

Reply:
Done